scenarios.md

```diff
@@ -194,6 +194,42 @@ A deployment requires a mydb image. The mydb image is routinely updated for secu
 1. The release management system deploys the new `mydb:1.0-202002131000` image.
 1. The production orchestrator validates it's signed with the Acme Rockets production key.
 
+
+
+### Scenario #7: Malicious and Erroneous Actions
+
+#### Scenario 7.1: A Repository Compromise Occurs
+An attacker manages to compromise a repository and gain access to the repository as an administrator.   
```

Comment:
This is a great scenario, but seems incomplete as it doesn't seem actionable as how would the state be known? It might just be the wording. Are you saying, the owner learns one or more repos have been compromised and needs to take action to re-secure it?

Reply:
Yes, this is my intent here.  I'm not sure I understand the distinction you are trying to make.  Can you propose an edit that clarifies?